src/components/throughput-widget/throughput-widget.tsx

```diff
@@ -73,18 +70,13 @@ export class ThroughputWidget {
   // Exchange ORCID bearer token for Throughput token.
   processOrcidAuthResponse() {
     const bearerToken = this.getFragmentParameterByName("access_token");
```

Comment:
Retrieve access token from query param. This access token is sent as a query param from the isamples server side.